tools/SetupFlow/DevHome.SetupFlow/Models/TaskProgress.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace DevHome.SetupFlow.Models;
+
+/// <summary>
+/// Represents the progress of a setup task.
+/// </summary>
+public sealed class TaskProgress
+{
+    public TaskProgress()

```

Comment:
Is the empty ctor meant to be called?  An empty constructor with no parameters is fishy.